css/main.css

```diff
@@ -1,11 +1,6 @@
-body {
-	padding-top: 3em;
-	max-width: 45em;
```

Comment:
Yes ! Our index.html really needs 3em. 
The compromise I was writing about is for the widget itself. 0 and 1em look very similar in any case.